node/core/approval-voting/src/backend.rs

```diff
@@ -148,7 +148,11 @@ impl<'a, B: 'a + Backend> OverlayedBackend<'a, B> {
 	// The assumption is that stored block range is only None on initialization.
 	// Therefore, there is no need to delete_stored_block_range.
```

Comment:
I think this comment needs to be updates, since `delete_stored_block_range` exists now